UQ-eLIPSE/prism-server
# bugfix/TEAM24-222

**If need setting up project:**
1. `./run_project_docker.sh` at prism root.
2.  Click Y for downloading mongodump data and if you need docker images built, click Y for that option too.
3. Run `./server/get_mongodumps.sh`
4. `mongorestore ./server/prism_mongodumps/agco360/agco360_mongodump_2024-01-18-v2/`
5. Navigate to `./server/env`
6. Change line 3:
```
DATABASE_URL=mongodb://mongodb:27017/agco360
```
7. Refer to **Setup**

**Setup:**
1. `git fetch`
2.`git checkout bugfix/TEAM24-222`
3. Change `./server/env` file to agco360 if haven't already:
```
DATABASE_URL=mongodb://mongodb:27017/agco360
```
4. `docker restart prism_server`
5. Run `localhost:3000/site`

**Testing:**
1. Please firstly check that all the "i" in the minimap nodes are rendered correctly and rooted upright in the minimap UI.
2. Make sure to enlarge the minimap, edit some nodes rotation, and ensure the "i" in the node is still upright.
3. `cd client`
4. Run `yarn test`, all tests should pass
5. Run `yarn cypress:run:local-agco360`, all tests besides the `MiniMapTest` should pass.

**Ticket:**
https://elipse-uq.atlassian.net/jira/software/c/projects/TEAM24/boards/150?selectedIssue=TEAM24-222
